content/resources/how-to-guides/how-to-zapier-alert.md

```diff
@@ -0,0 +1,117 @@
+---
+title: How to Alert Task to Zapier Webhook
+description: >
+  Learn how to use webhooks with InfluxDB to send alerts to Zapier
+menu:
+  resources:
+    parent: How-to guides
+weight: 102
+---
+### How To: Alert Task to Zapier Webhook
+
+### By: Tyler Nguyen
```

Comment:
- Docs shouldn't start with titles. The `title` in the frontmatter is used as the page h1
- Don't include author information in docs

```suggestion
```